aspnetcore/fundamentals/tools/dotnet-aspnet-codegenerator.md

```diff
@@ -1,42 +1,39 @@
 ---
-title: dotnet aspnet-codegenerator command
+title: ASP.NET Core code generator tool (`aspnet-codegenerator`)
 author: tdykstra
-description: The dotnet aspnet-codegenerator command scaffolds ASP.NET Core projects.
+description: The ASP.NET Core code generator tool scaffolds ASP.NET Core projects.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: tdykstra
-ms.date: 12/05/2022
+ms.date: 06/17/2024
 uid: fundamentals/tools/dotnet-aspnet-codegenerator
 ---
+# ASP.NET Core code generator tool (`aspnet-codegenerator`)
 
-# dotnet aspnet-codegenerator
+The `dotnet aspnet-codegenerator` command runs the ASP.NET Core scaffolding engine. Executing the `dotnet aspnet-codegenerator` command is only required to scaffold from the command line or when using Visual Studio Code. The command isn't required to use scaffolding with Visual Studio, which includes the scaffolding engine by default.
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT)
-
-`dotnet aspnet-codegenerator` - Runs the ASP.NET Core scaffolding engine. `dotnet aspnet-codegenerator` is only required to scaffold from the command line, it's not needed to use scaffolding with Visual Studio.
-
-## Install and update aspnet-codegenerator
+## Install and update the code generator tool
 
 Install the [.NET SDK](https://dotnet.microsoft.com/download).
 
-`dotnet aspnet-codegenerator` is a [global tool](/dotnet/core/tools/global-tools) that must be installed. The following command installs the latest stable version of the `dotnet aspnet-codegenerator` tool:
+`dotnet aspnet-codegenerator` is a [global tool](/dotnet/core/tools/global-tools) that must be installed. The following command installs the latest stable version of the ASP.NET Core code generator tool:
 
 ```dotnetcli
 dotnet tool install -g dotnet-aspnet-codegenerator
 ```
 
 [!INCLUDE[](~/includes/dotnet-tool-install-arch-options.md)]
 
-The following command updates `dotnet aspnet-codegenerator` to the latest stable version available from the installed .NET Core SDKs:
+If the tool is already installed, the following command updates the tool to the latest stable version available from the installed .NET Core SDKs:
 
 ```dotnetcli
 dotnet tool update -g dotnet-aspnet-codegenerator
 ```
 
-## Uninstall aspnet-codegenerator
+## Uninstall the code generator tool
 
-It may be necessary to uninstall the `aspnet-codegenerator` to resolve problems. For example, if you installed a preview version of `aspnet-codegenerator`, uninstall it before installing the released version.
+It may be necessary to uninstall the ASP.NET Core code generator tool to resolve problems. For example, if you installed a preview version of the tool, uninstall it before installing the released version.
```

Comment:
This is the guidance that Rick had in place. In the Blazor tutorial, I think Dan and I will settle only on having devs re-run the install command because it updates automatically. We don't envision devs working that tutorial with preview tooling, and we even instruct devs not to do so in the tutorial. This might be here because there's a special case with preview tooling installs that only this process will address.